pages/twoplayer/index.js

```diff
@@ -1,14 +1,28 @@
 import StyledContainer from "../../components/StyledContainer";
 import StyledInfoSpan from "../../components/StyledInfoSpan";
 import TwoPlayerGrid from "../../components/TwoPlayerGrid";
+import StyledDoubleSection from "../../components/StyledDoubleSection";
 import store from "../../zustand/store";
 
 export default function TwoPlayerPage() {
   const player = store((state) => state.player);
+  const matchesPlayerOne = store((state) => state.matchesPlayerOne);
+  const matchesPlayerTwo = store((state) => state.matchesPlayerTwo);
+
+  console.log(matchesPlayerOne, matchesPlayerTwo);
```

Comment:
I think we would want to remove console logs before merging into main.

Reply:
👍 thanks